app/main/views/briefs.py

```diff
@@ -204,6 +206,113 @@ def update_brief_status(brief_id, action):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/award', methods=['POST'])
+def award_pending_brief_response(brief_id):
+    json_payload = get_json_from_request()
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if not brief.status == 'closed':
+        abort(400, "Brief is not closed")
+
+    brief_responses = BriefResponse.query.filter(
+        BriefResponse.brief_id == brief.id,
+        BriefResponse.status != 'draft'
+    )
+    # Check that BriefResponse in POST data is associated with this brief
+    if not json_payload['brief_response_id'] in [b.id for b in brief_responses]:
+        abort(400, "BriefResponse cannot be awarded for this Brief")
+
+    # Find any existing pending awarded BriefResponse
+    audit_events = []
+    pending_awarded_responses = list(filter(lambda x: x.status == 'pending-awarded', brief_responses))
+    if pending_awarded_responses:
```

Comment:
Feels like a for loop here would indicate what we're trying to do better.
_If_ there was more than one `pending_awarded_response` we'd want to blank all of them. Also calling the variable `pending_awarded_responses` and then treating it as if there's only going to ever be one (`pending_awarded_responses[0]`) feels a little semantically weird :man_shrugging: